src/emulator/gui/src/game_selector.cpp

```diff
@@ -179,22 +181,98 @@ void draw_game_selector(GuiState &gui, HostState &host) {
                     }
                     if (ImGui::IsMouseClicked(0))
                         selected[0] = true;
+                    if (gui.title_id != game.title_id)
+                        gui.title_id = game.title_id;
                 }
+
+                if (gui.title_id == game.title_id) {
+                    if (ImGui::BeginPopupContextItem("#game_context_menu")) {
+                        if (ImGui::MenuItem("Boot", game.title.c_str()))
+                            selected[0] = true;
+                        if (ImGui::BeginMenu("Copy Game Info")) {
+                            if (ImGui::MenuItem("ID and Name")) {
+                                ImGui::LogToClipboard();
+                                ImGui::LogText("%s [%s]", game.title_id.c_str(), game.title.c_str());
+                                ImGui::LogFinish();
+                            }
+                            if (ImGui::MenuItem("ID")) {
+                                ImGui::LogToClipboard();
+                                ImGui::LogText("%s", game.title_id.c_str());
+                                ImGui::LogFinish();
+                            }
+                            if (ImGui::MenuItem("Name")) {
+                                ImGui::LogToClipboard();
+                                ImGui::LogText("%s", game.title.c_str());
+                                ImGui::LogFinish();
+                            }
+                            ImGui::EndMenu();
+                        }
+                        if (ImGui::BeginMenu("Remove")) {
+                            if (ImGui::MenuItem("Game"))
+                                gui.delete_game_popup = true;
+                            if (ImGui::MenuItem("Shader Log")) {
+                                if (fs::exists(shaderlog_path / game.title_id))
+                                    fs::remove_all(shaderlog_path / game.title_id);
+                            }
+                            ImGui::EndMenu();
+                        }
+                        ImGui::EndPopup();
+                    }
+
+                    auto games_index = std::find_if(gui.game_selector.games.begin(), gui.game_selector.games.end(), [game](Game g) {
```

Comment:
```suggestion
                    auto games_index = std::find_if(gui.game_selector.games.begin(), gui.game_selector.games.end(), [&game](const Game &g) {
```

Don't do unnecessary copy

Reply:
I also think we only should find it when needed. 256 games can make this a bottleneck.

Reply:
i have tested on my main instal with 500 game/hb have works, and have delete few game (10) without probleme